cargo-near/src/util/tracking.rs

```diff
@@ -0,0 +1,47 @@
+use chrono::prelude::*;
+use reqwest::Client;
+use rustc_version_runtime::version;
+use std::env;
+
+#[derive(Debug, serde::Serialize)]
+struct MixpanelProperties {
+  token: String,
+  rustc_version: String,
+  os: String,
+  timestamp: String,
+}
+
+#[derive(Debug, serde::Serialize)]
+struct TrackingData {
+    event: String,
+    properties: MixpanelProperties,
+}
+
+pub(crate) async fn track_usage() {
```

Comment:
Do we have to check CI env here?
Like we do in `create-near-app` [here](https://github.com/near/create-near-app/blob/9c482fd0871a206fae04dabc3e91fb775d9b88f9/src/tracking.ts#L14)

Because I didn't find any tests for the `cargo near new` command in workflows

Reply:
It won't contribute that much to the total usage of the tool, so I would skip that.

Reply:
@frol, well, doing a test in CI  to check this specific `track_usage` call result is ok, i.e. that verifying that call was configured properly  and was recorded on some usage tracking remote, sounds like a good idea to me

Reply:
Yeah, having a test is useful, but that is not what @garikbesson asked and linked in create-near-app